### PR TITLE
Update the version description in auto random document

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)
Update the description of `AUTO RANDOM` feature.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3749
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: auto-random.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ aliases: ['/docs/dev/auto-random/','/docs/dev/reference/sql/attributes/auto-rand
 
 > **Warning:**
 >
-> `AUTO_RANDOM` is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment. In later TiDB versions, the syntax or semantics of `AUTO_RANDOM` might change.
+> `AUTO_RANDOM` has been an official feature since v4.0.3. It is **recommended** that you use this attribute in the production environment. In later TiDB versions, the syntax or semantics of `AUTO_RANDOM` might change.
```
Comment (nullnotnil) [inline auto-random.md:11]: It doesn't make sense that the syntax might change if it is now stable. May I suggest:
```suggestion
> `AUTO_RANDOM` was marked as stable in v4.0.3.
```

File: auto-random.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ aliases: ['/docs/dev/auto-random/','/docs/dev/reference/sql/attributes/auto-rand
 
 > **Warning:**
```
Comment (nullnotnil) [inline auto-random.md:9]: This no longer needs to be a warning. It is stable now.
```suggestion
> **Note:**
```

File: auto-random.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ aliases: ['/docs/dev/auto-random/','/docs/dev/reference/sql/attributes/auto-rand
 
 # AUTO_RANDOM <span class="version-mark">New in v3.1.0</span>
 
-> **Warning:**
+> **Note:**
 >
-> `AUTO_RANDOM` is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment. In later TiDB versions, the syntax or semantics of `AUTO_RANDOM` might change.
+> `AUTO_RANDOM` was marked as stable in v4.0.3.
```
Comment (TomShawn) [inline auto-random.md:11]:
```suggestion
> `AUTO_RANDOM` was marked as stable in v4.0.3. Now you can use it in the production environment. 
```

**Comment by nullnotnil** [comment]
LGTM

**Pull request merged**